assets/airports/kcvg.json

```diff
@@ -0,0 +1,6785 @@
+{
+    "radio": {
+        "twr": "Cincinnati Tower",
+        "app": "Cincinnati Approach",
+        "dep": "Cincinnati Departure"
+    },
+    "icao": "KCVG",
+    "iata": "CVG",
+    "magnetic_north": -4,
```

Comment:
Magnetic variation is -5.6
![var](https://cloud.githubusercontent.com/assets/13125311/25864465/6be17880-34e7-11e7-966d-0504956bb927.PNG)